internal/planner/planner.go

```diff
@@ -683,6 +700,7 @@ func (p *Planner) planExprCall(e *ast.Expr, iter planiter) error {
 		})
 	case ast.GreaterThan.Name:
 		return p.planBinaryExpr(e, func(a, b ir.Local) error {
+			p.loc = ploc // XXX: exemplary, all of those above and below would need this
```

Comment:
Is this related to providing term-level locations? I'm just wondering why we need this.

Reply:
Yeah will cleanup. 🙈